GetMerchant/__tests__/handler.test.ts

```diff
@@ -0,0 +1,255 @@
+/* tslint:disable: no-any */
+
+import { ProductCategoryEnum } from "../../generated/definitions/ProductCategory";
+import { ProductCategoryEnumModelType } from "../../models/ProductCategories";
+import { GetMerchantHandler } from "../handler";
+
+const queryMock = jest.fn().mockImplementation(_ => {});
+
+const cgnOperatorDbMock = { query: queryMock };
+
+beforeEach(() => {
+  jest.clearAllMocks();
+});
+
+describe("GetMerchantHandler", () => {
```

Comment:
In order to improve readability we can move constants definition outside the `describe` section :)